SpatialGDK/Documentation/content/glossary.md

```diff
@@ -208,6 +208,22 @@ The Launcher downloads the client executable from the [SpatialOS assembly](#asse
 >
 > [The Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher)
 
+### Layers
+
+In SpatialOS, you can split up [server-worker](#workers) computation into layers, with each layer of server-worker instances handling a specific and unique aspect of your game.
+
+A SpatialOS layer has two elements;
+
+* a group of SpatialOS entity components,
```

Comment:
I'd argue this is ambiguous between how layers work and how ACLs worked. A layer (or at least component group or whatever we end up using) is intrinsic to the component, however here this could be read as a group of components on existing entities, like an ACL requirement.

Reply:
so "a group of SpatialOS component definitions"?

Reply:
Yes. I think that would be better for now.